java/flight/flight-sql/src/test/java/org/apache/arrow/flight/sql/example/FlightSqlExample.java

```diff
@@ -433,6 +483,52 @@ private static VectorSchemaRoot getRoot(final ResultSet data, final BufferAlloca
     return new VectorSchemaRoot(singletonList(dataVector));
   }
 
+  private static VectorSchemaRoot getTypeInfoRoot(CommandGetXdbcTypeInfo request, ResultSet typeInfo,
+                                                  final BufferAllocator allocator)
+      throws SQLException {
+    Objects.requireNonNull(allocator, "BufferAllocator cannot be null.");
```

Comment:
I think Preconditions is how we usually validate this.

Reply:
Fixed